vuepress/docs/next/tutorials/create/ms/generate-microservices-and-micro-frontends.md

```diff
@@ -1,81 +1,67 @@
----
-sidebarDepth: 2
----
-
 # Generate Microservices and Micro Frontends
 
 ## Overview
 
-This tutorial describes how to use the Entando Component Generator (ECG) to create microservices and micro frontends for deployment to the [Entando Component Repository](../../../docs/compose/local-hub-overview.md) and Entando Applications. The ECG is powered by [JHipster](https://www.jhipster.tech/) and leverages the Entando Blueprint. 
-
-![Entando Component Generator](./img/component-gen-flow.png)
+This tutorial describes how to use the Entando Component Generator (ECG) to create microservices and micro frontends for deployment to the [Local Hub](../../../docs/compose/local-hub-overview.md) of an Entando Application or a shared [Entando Hub](../../../docs/getting-started/landing-page.md#entando-hub). The ECG is powered by [JHipster](https://www.jhipster.tech/) and leverages the Entando Blueprint.
 
-The general flow of component generation is:
-
-1. Run the Entando Blueprint to create your components (Spring Boot microservice and optionally React micro frontends)
-2. Customize and enhance your generated code
-3. Build an Entando Bundle from your components
-4. Deploy a [custom resource](../../../docs/consume/custom-resources.md) for your bundle into Kubernetes
-5. Install your Entando Bundle into your Entando Application(s)
+The output of this tutorial is [a new bundle project](../../../docs/curate/bundle-details.md#bundle-development-process) with several components: 
```

Comment:
the first word of each bulleted line item below should be capitalized (or use 3 for the second bullet)